lib/secure_headers/railtie.rb

```diff
@@ -3,9 +3,22 @@
   module SecureHeaders
     class Railtie < Rails::Engine
       isolate_namespace ::SecureHeaders if defined? isolate_namespace # rails 3.0
+      conflicting_headers = ['X-Frame-Options', 'X-XSS-Protection', 'X-Content-Type-Options',
+                             'X-Permitted-Cross-Domain-Policies', 'X-Download-Options',
+                             'X-Content-Type-Options', 'Strict-Transport-Security',
+                             'Content-Security-Policy', 'Content-Security-Policy-Report-Only',
+                             'X-Permitted-Cross-Domain-Policies']
```

Comment:
Yeah, each header class does have a name constant, but with CSP it's a little awkward. I think the string values are fine.

Please add the hpkp headers too twitter/secureheaders/pull/132